This is so that we can pass the command argument with spaces.
Note that this is not really portable, $@ is a bashism, and
traditional shell does not have a suitable replacement.

Lubomir Rintel <lkundrak@v3.sk>

--- kBuild-0.1.3/kBuild/env.sh.escape	2008-08-17 21:13:20.000000000 +0200
+++ kBuild-0.1.3/kBuild/env.sh	2008-08-17 21:13:23.000000000 +0200
@@ -568,7 +568,7 @@
             MY_RC=$?
         else
             test -z "${QUIET_OPT}" && echo "$0: info: Executing command: $*" 1>&${ERR_REDIR}
-            $*
+            "$@"
             MY_RC=$?
             test -z "${QUIET_OPT}" -a "$MY_RC" -ne 0 && echo "$0: info: rc=$MY_RC: $*" 1>&${ERR_REDIR}
         fi
